packages/next-css/readme.md

```diff
@@ -137,28 +140,6 @@ Your exported HTML will then reflect locally scoped CSS class names.
 
 For a list of supported options, [refer to the webpack `css-loader` README](https://github.com/webpack-contrib/css-loader#options).
 
-```js
```

Comment:
Seems like this was a duplication? It's the same as the code example in the usage section and didn't seem like it is related to the content in line 138.